### PR TITLE
Adding pyart_mch package

### PR DESCRIPTION
**Pull request by meteoswiss-mdr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by meteoswiss-mdr** [comment]
Can anybody help me making sense of these errors?
I am new to this business. I tested the packages locally and there was no problem.

**Comment by marcelotrevisani** [comment]
It showing some ``InvalidSyntax``
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=123796&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=3097

**Comment by marcelotrevisani** [comment]
If you want to support Python 2, you need to add
```
from __future__ import print_function
```
in those files which you are using ``print("", file=...)``

**Comment by marcelotrevisani** [comment]
But you can ``skip`` python 2 if you want to

**Comment by meteoswiss-mdr** [comment]
Thanks for your help. I want to skip python 2. How can I tell so? Should I specify it in the recipe?

**Comment by marcelotrevisani** [comment]
> Thanks for your help. I want to skip python 2. How can I tell so? Should I specify it in the recipe?

Yes, you can add a section inside of build like that
```
build:
  skip: true  #[py2k]
```

File: recipes/pyart_mch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/pyart_mch/meta.yaml]:
```suggestion
  number: 0
  skip: true  # [py2k]
```

File: recipes/pyart_mch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  script:  # [not win]
```
Comment (marcelotrevisani) [inline recipes/pyart_mch/meta.yaml]: if you want to skip windows you can do it using the ``skip`` as well

File: recipes/pyart_mch/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k]
```
Comment (marcelotrevisani) [inline recipes/pyart_mch/meta.yaml]:
```suggestion
  skip: true  # [py2k or win]
```

File: recipes/pyart_mch/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k or win]
+  script:  # [not win]
```
Comment (marcelotrevisani) [inline recipes/pyart_mch/meta.yaml]:
```suggestion
  script:
```

File: recipes/pyart_mch/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/pyart_mch/meta.yaml]: as you are skipping windows now, you can remove the selectors ``# [not win]`` from the recipe :)

File: recipes/pyart_mch/meta.yaml (hunk 4 above)
Comment (meteoswiss-mdr) [reply recipes/pyart_mch/meta.yaml]: OK. I hope it will pass now. Thanks a lot

**Comment by meteoswiss-mdr** [comment]
appveyor was not successful. Does it need to be? What is next?

**Comment by marcelotrevisani** [comment]
> appveyor was not successful. Does it need to be? What is next?

Nope, that is fine! :)

**Comment by marcelotrevisani** [comment]
It may take about one hour to create the feedstock and have the package available on ``conda-forge`` channel
Thanks for adding this project! :tada: :tada: :tada:

**Comment by meteoswiss-mdr** [comment]
Great! That was fast! Thanks a lot for your support!

**Pull request merged**